### ISSUE TITLE
MultilineRawStringIndentation allowEmptyLines/strictIndentation flag

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#5810]
I'll use `.` instead of space in indentation so we can see what we're talking about.

## Expected Behavior of the rule

empty line = lines that don't contain characters other than space or tab.

New configuration options: `emptyLines`, values:
 * `allow`: empty lines can indented with `0` or `[indentSize * indentLevel, ∞)` spaces.   
   Current behavior.
 * `indent`: empty lines must be indented at least with `indentSize * indentLevel` spaces.  
   What was discussed in https://github.com/detekt/detekt/pull/5804#pullrequestreview-1308732242.
 * `strict`: empty lines must be indented with exactly `indentSize * indentLevel` spaces, so indentation must match the indentation of the whole code block, no more, no less.  
   This option would help to prevent "trailing spaces" INSIDE triple quotes on empty lines, e.g.
   ```
   ....val x = """
   ........some
   ..........
   ........thing
   ....""".trimIndent()
   ```

### allow
```yaml
MultilineRawStringIndentation:
    emptyLines: allow
```

```xml
<compliant>
....val x = """
........fun f() {

........}
....""".trimIndent()

....val x = """
........fun f() {
........
........}
....""".trimIndent()


....val x = """
........fun f() {
..........
........}
....""".trimIndent()
</compliant>
```

### indent
```yaml
MultilineRawStringIndentation:
    emptyLines: indent
```

```xml
<compliant>
....val x = """
........fun f() {
........
........}
....""".trimIndent()

....val x = """
........fun f() {
..........
........}
....""".trimIndent()
</compliant>

<noncompliant>
....val x = """
........fun f() {

........}
....""".trimIndent()
</noncompliant>
```

### strict

```yaml
MultilineRawStringIndentation:
    emptyLines: strict
```

```xml
<compliant>
....val x = """
........fun f() {
........
........}
....""".trimIndent()
</compliant>

<noncompliant>
....val x = """
........fun f() {

........}
....""".trimIndent()

....val x = """
........fun f() {
..........
........}
....""".trimIndent()
</noncompliant>
```

## Context
See https://github.com/detekt/detekt/pull/5804#pullrequestreview-1308732242

**Comment by BraisGabin** [comment]
I'm not 100% sure about `strict`. I think that check should be moved to a rule about trailing spaces inside strings. It's out of scope of `MultilineRawStringIndentation`. Why this rule should care about:

```
........fun f() {
..........
........}
```

but should not about
```
........fun f() {...
........
........}
```

I define empty as nothing at all. A line with spaces I would call it `blank` instead of `empty`.

Then I would add a flag called `allowEmptyLines`:
- `true` would map to `allow`
- `false` would map to `indent`

I like the enum idea but the possible values of an enum are difficult to find in a `yaml`. (This remembers me that we still have #4162 open)
